chrome-extension/manifest.json

```diff
@@ -15,14 +15,21 @@
   "web_accessible_resources": [
     "assets/*"
   ],
+  "background": {
+    "scripts": ["background.js"],
+    "persistent": true
+    },
   "permissions": [
     "https://*/*",
     "http://*/*",
     "tabs",
     "webRequest",
     "webRequestBlocking",
-    "storage",
-    "activeTab",
-    "contextMenus"
-  ]
+    "identity",
+    "identity.email"
+  ],
+  "oauth2": {
+    "client_id": "37652710111-9i14b191phogsofqh3cn99fripb8brpf.apps.googleusercontent.com",
```

Comment:
This should not be checked in to the repository. I see some other comments about changing behavior based on whether the chrome extension is in dev or prod. [This](https://developer.chrome.com/extensions/management#method-get) may be useful to you. You can use that method to get an  [ExtensionInstallType](https://developer.chrome.com/extensions/management#type-ExtensionInstallType)

Reply:
oh actually i forgot to delete that. I don't think im using it p

Reply:
Oh okay cool!